### ISSUE TITLE
feat: Add label support for the <FluentAutocomplete>

### ISSUE THREAD

**Issue by vejnoe** [microsoft/fluentui-blazor#812]
# 🙋 Feature Request

Would like the option to add a label to the <FluentAutocomplete> component. I can see it is a form of wrapper to the <FluentTextInput> component so would be nice if you could add a label just by adding the text inside the component.

## 🤔 Expected Behavior

You should be able to to someting like:
```
<FluentAutocomplete ...>
   Search tags
</FluentAutocomplete>
```

## 😯 Current Behavior

It's just ignoring <ChildContent> and there is not label attribute.

## 💁 Possible Solution

The <FluentTextInput> already have this feature, so are imagining that the <FluentAutocomplete> could used a <ChildContent> to pass on the label text to the <FluentTextInput>'s <ChildContent>...

## 🔦 Context

My issue is the styling, I cant reuse the <label> styling by applying the class .label to my own label. Since its styled via the JS some how. And then I feel like it would make sense that I could use the same approach as on a normal input field.

**Comment by dvoituron** [comment]
Thank you for this request. It makes sense to have the same logic used by `FluentTextField`.
I'll add it in the next version.

**Comment by dvoituron** [comment]
> Thank you for this request. It makes sense to have the same logic used by `FluentTextField`. I'll add it in the next version.

We'll add a `Label` attribute (of type `string`) to manage this information. Not only on this `Autocomplete` component, but also on the other "Input" components. This will enable us to correctly manage the label to be displayed, but also to make it compatible with the Accessibility rules.

**Comment by vejnoe** [comment]
> We'll add a `Label` attribute (of type `string`) to manage this information. Not only on this `Autocomplete` component, but also on the other "Input" components. This will enable us to correctly manage the label to be displayed, but also to make it compatible with the Accessibility rules.

Will this mean that we can not add html element like icons and span to the <label>? If so that would be sad, that was one of my favoured features with Fluent UI compared to MudBlazor...

**Comment by dvoituron** [comment]
Good question.
I could add a `Label` attribute for most cases.
And a `LabelTemplate` property (`RenderFragment`) for special cases.
But then you'd have to use `Title` to define the `aria-label` content of the Accessibility.

**Comment by vejnoe** [comment]
> Good question. I could add a `Label` attribute for most cases. And a `LabelTemplate` property (`RenderFragment`) for special cases. But then you'd have to use `Title` to define the `aria-label` content of the Accessibility.

Really like the idea of a LabelTemplate, but all cool that you would only need Label in most cases.

**Comment by dvoituron** [comment]
Fixed in #822
